mmv1/products/privateca/terraform.yaml

```diff
@@ -68,17 +45,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           ~> **Note:** For `SUBORDINATE` Certificate Authorities, they need to
           be manually activated (via Cloud Console of `gcloud`) before they can
           issue certificates.
-      config.reusableConfig.reusableConfig: !ruby/object:Overrides::Terraform::PropertyOverride
-        description: |
-          {{description}}. Alternatively, one of the short names
-          found by running `gcloud beta privateca reusable-configs list`.
-        diff_suppress_func: 'certificateAuthorityReusableConfigDiffSuppress'
+      config.x509Config: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_flatten: 'templates/terraform/custom_flatten/privateca_certificate_509_config.go.erb'
     custom_code: !ruby/object:Provider::Terraform::CustomCode
-      constants: templates/terraform/constants/certificate_authority.go.erb
-      encoder: templates/terraform/encoders/certificate_authority.go.erb
-      pre_delete: templates/terraform/pre_delete/privateca_certificate_authority.go.erb
+      decoder: templates/terraform/decoders/treat_deleted_state_as_gone.go.erb
+      post_create: templates/terraform/post_create/privateca_authority_enable.go.erb
```

Comment:
That is a little complicated as it would require manual steps outside of Terraform to enable the CA, or somewhat complicated state management within Terraform itself. It should be possible in the future to enable this via Terraform, but I believe this is a reasonable default behavior at the moment